spec/factories/books.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :book do
+    title "MyString"
```

Comment:
I just removed the files for now. I am working in new files on a different branch and I am using Faker in those.